src/dataviews/dataview-model-base.js

```diff
@@ -322,6 +322,10 @@ module.exports = Model.extend({
     return sourceAnalysis && sourceAnalysis.get('type');
   },
 
+  getLayerName: function () {
+    return this.layer.get('layer_name');
```

Comment:
I guess we could also return `''` here  if there's no layer. Not sure if it has any advantage though...

```
if (this.layer) {
  return this.layer.get('layer_name');
}

return '';
```